packages/printer-legacy/src/directive.js

```diff
@@ -0,0 +1,85 @@
+const {
+  graphql: { getDirective },
+  object: { hasProperty },
+} = require("@graphql-markdown/utils");
+const {
+  HEADER_SECTION_LEVEL,
+  HEADER_SECTION_SUB_LEVEL,
+  MARKDOWN_EOP,
+  MARKDOWN_EOL,
+} = require("./const/strings");
+const { printLink } = require("./link");
+
+function getConstDirectiveMap(type, options) {
```

Comment:
I'd say that `getConstDirectiveMap` should belong to `@graphql-markdown/utils` since it is pure GQL schema manipulation and the method may be reused for other purpose outside printer.